source/server.js

```diff
@@ -44,14 +47,14 @@ function validateEvent(req, res, next) {
 }
 
 app.get('/', function (req, res) {
-	res.json({app: 'notifier', env: process.env.NODE_ENV, version: package.version, apiUrl: '/api'});
+	res.json({ app: 'notifier', env: process.env.NODE_ENV, version: package.version, apiUrl: '/api' });
 });
 
 app.post('/api/events', checkAccessToken, validateEvent, function (req, res) {
 	var e = req.body;
-	bus.publish(e.event, {event: e});
+	bus.publish(e.event, { event: e });
 
-	res.send(201);
+	res.status(201).send("Created");
```

Comment:
It's not that important, but better that API responds some json. In this case, I would just return sent event.